src/builders/anndata_builders.py

```diff
@@ -34,11 +36,18 @@ def get_conf_cells(self):
         # or AnnData default (like X_umap or X).
         cell_set_obs = ["leiden"]
         cell_set_obs_names = ["Leiden"]
-        dags = [dag
-                for dag in self._entity['metadata']['dag_provenance_list'] if 'name' in dag]
+        dags = [
+            dag for dag in self._entity['metadata']['dag_provenance_list']
+            if 'name' in dag]
         if(any(['azimuth-annotate' in dag['origin'] for dag in dags])):
-            cell_set_obs.append("predicted.ASCT.celltype")
-            cell_set_obs_names.append("Predicted ASCT Cell Type")
+            request_init = self._get_request_init() or {}
+            headers = request_init.get('headers', {})
+            response = requests.get(
+                f'{adata_url}/uns/annotation_metadata/is_annotated/0',
+                headers=headers)
+            if response.content == b'\x01':
```

Comment:
I share your confusion, Chuck. Maybe add a comment to explain the significance of `b'\x01'`?

Reply:
I'll file an issue. Thanks for the nudge.